packages/mobile/src/account/reducer.ts

```diff
@@ -111,10 +112,14 @@ export const reducer = (
       }
     case Actions.DEV_MODE_TRIGGER_CLICKED:
       const newClickCount = (state.devModeClickCount + 1) % 6
+      const devModeActive = newClickCount >= 3
+      if (devModeActive) {
+        Logger.showMessage('Debug Mode Activated')
+      }
```

Comment:
nit: though this is harmless here, reducers shouldn't have any side effect.
This should be at the component level or saga.

Reply:
I considered that but that would require us to create a watcher for this event which duplicates the logic in this reducer. Just not worth it for 'purity' in this case I think